Various fixes. Mainly changing relative links to absolute links

Graeme-Miller opened the pull request:
Changed some relative links to absolute links.

I have run a link checker on the docs with and without these changes. Without these changes there is one more broken link.

drigodwin commented on guide/ops/production-installation.md:
```diff
@@ -26,7 +26,7 @@ Check that the server meets the [requirements](requirements.html).
 
 * install Java JRE or JDK (version 7 or later)
-* install an [SSH key]({{ site.path.guide }}/ops/locations/index.html#ssh-keys), if not available
+* install an [SSH key]({{ site.path.guide }}/ops/locations#ssh-keys), if not available
```
Why remove this index.html but not the one on the line below?

drigodwin commented on guide/yaml/winrm/index.md:
```diff
@@ -402,7 +402,7 @@ on the Microsoft TechNet site.
 ---------------
 
-Much of the [operations troubleshooting guide](../../ops/troubleshooting/) is applicable for Windows blueprints.  
+Much of the [operations troubleshooting guide](../{{ site.path.guide }}/ops/troubleshooting/) is applicable for Windows blueprints.  
```
This shouldn't have a `..` at the start

Graeme-Miller commented:
I am going to make the changes suggested by Duncan to the branch- do not merge yet as I will be testing these changes (I need them pushed so that amp-docs can build correctly)

Graeme-Miller commented:
Ok, I have tested with the fixes and it looks good. I have removed the production-installation changes, as they are not required in the downstream project any more and I have fixed the accidental relative link.

@aledsage can you merge this?

aledsage commented:
LGTM; merging now.
